features/malicious-site-protection.json

```diff
@@ -0,0 +1,8 @@
+{
+    "_meta": {
+        "description": "Malicious Site Protection",
```

Comment:
Should we explain the rename in the phishing protection feature (aka it's deprecated)